backend/app/user/routes.py

```
@@ -7,13 +7,16 @@
     UserProfileUpdateRequest,
 )
 from app.user.service import user_service
-from fastapi import APIRouter, Depends, HTTPException, status
+from fastapi import APIRouter, Depends, HTTPException, Request, status
```

Ruff (0.12.2)

`fastapi.status` imported but unused

Remove unused import: `fastapi.status`

(F401)